tests/filter.json

```diff
@@ -866,6 +901,91 @@
         {"a": 0.011, "d": "e"}
       ]
     },
+    {
+      "name": "equals number, invalid plus",
+      "selector" : "$[?@.a==+1]",
+      "invalid_selector": true
+    },
+    {
+      "name": "equals number, invalid minus space",
+      "selector" : "$[?@.a==- 1]",
+      "invalid_selector": true
+    },
+    {
+      "name": "equals number, invalid double minus",
+      "selector" : "$[?@.a==--1]",
```

Comment:
This one could be interesting if/when we add math operators.

Reply:
Good point; but for now should this be kept? If necessary it can be removed / adjusted later in case this becomes valid.

Reply:
IMO it's a good test.

Reply:
Yes, just a comment.